src/main/webapp/electionInfo.html

```diff
@@ -154,6 +154,12 @@ <h3 id="info-cards-title">What's on the ballot:</h3>
                               {{/if}}
                               {{#each this.channels}}
                               <p><b>{{@key}}: </b><a href={{this}}>{{stripUrl this}}</a></p>
+                              {{else}}
+                              {{#unless this.campaignSite}}
```

Comment:
I'm not really familiar with handlebars, is it possible to combine these two #unless blocks?

Reply:
I tried looking this up, and found there's no built-in way to do it: https://stackoverflow.com/questions/41251635/handlebars-multiple-conditions-if-statement

But I can make a helper "and" operator which gets the job done the same.